scripts/mdbook-forc-documenter/examples/forc_completions.md

```diff
@@ -100,7 +20,7 @@ Run the command:
 
 ```sh
 mkdir -p ~/.local/share/bash-completion/completions
-forc completions --shell=bash >> ~/.local/share/bash-completion/completions/forc
+forc competions --shell=bash >> ~/.local/share/bash-completion/completions/forc
```

Comment:
Hmmm
```suggestion
forc completions --shell=bash >> ~/.local/share/bash-completion/completions/forc
```

Reply:
Oopsie, probably accidentally removed it while copying the whole chunk over